include/nix/Dimensions.hpp

```diff
@@ -624,14 +624,28 @@ class NIXAPI RangeDimension : public base::ImplContainer<base::IRangeDimension>
      * values. This can be used to store data that is sampled at irregular
      * intervals.
      *
-     * Ticks must be ordered in ascending order.
-     *
      * @return A vector with all ticks for the dimension.
      */
     std::vector<double> ticks() const {
         return backend()->ticks();
     }
 
+    /**
+     * @brief Get a number of ticks from the dimension.
+     *
+     * The ticks map the index of the data at the respective dimension to other
+     * values. This can be used to store data that is sampled at irregular
+     * intervals.
+     *
+     * @param start size_t the start index.
+     * @param count size_t the number of ticks to read.
+     *
+     * @return A vector with the ticks for the dimension.
+     */
+    std::vector<double> ticks(size_t start, size_t count) const {
```

Comment:
A dimension can be of `ndsize_t`, right? So start could also be of `ndsize_t`. In theory `count` could to, but then it would not fit into `std::vector` (and maybe the RAM), so I think for `count` it is fine.

Reply:
yes, that was my thinking, std::vector always uses size_t